File: funciones/1.php
<?php

declare(strict_types=1);

//inicio funciones");

function maximo(){

    $num1 = readline("Digite su primer número: ");

    $num2 = readline("Digite su segundo número: ");
    
    $num3 =  readline("Digite su tercer número: ");

    if($num1 >= $num2 && $num2 >= $num3){ 
        echo "Este es el número máximo de los tres: 
        {$num1}";

    }else if($num1 <= $num2 && $num1 >= $num3){ 
        echo "Este es el número máximo de los tres: 
        {$num2}";
    
    }else if($num1 <= $num2 && $num2 <= $num3){ 
        echo "Este es el número máximo de los tres: 
        {$num3}";
    }else if($num1 >= $num2 && $num2 <= $num3 && $num1 >= $num3){ 
        echo "Este es el número máximo de los tres: 
        {$num1}";
    }else if($num1 >= $num2 && $num3 >= $num1){ 
        echo "Este es el número máximo de los tres: 
        {$num3}";
    }else if ($num1 >= $num2 && $num2 <= $num3 && $num1 >= $num3){
        echo "Este es el número máximo de los tres: 
        {$num1}";
    }

}

maximo();

?>

File: dart/funciones/1.dart
import 'dart:io';

 void maximo(){
     
     print("Digite su primer número: ");

    int num1 = int.parse(stdin.readLineSync()!);
    
    print("Digite su segundo número: ");

    int num2 = int.parse(stdin.readLineSync()!);
    
    print("Digite su tercer número: ");
    
    int num3 = int.parse(stdin.readLineSync()!);

    if(num1 >= num2 && num2 >= num3){ 
         print("Este es el número máximo de los tres: ${num1}");

    }else if(num1 <= num2 && num1 >= num3){ 
         print("Este es el número máximo de los tres: ${num2}");
    
    }else if(num1 <= num2 && num2 <= num3){ 
         print("Este es el número máximo de los tres: ${num3}");
    }else if(num1 >= num2 && num2 <= num3 && num1 >= num3){ 
         print("Este es el número máximo de los tres: ${num1}");
    }else if(num1 >= num2 && num3 >= num1){ 
         print("Este es el número máximo de los tres: ${num3}");
    }else if (num1 >= num2 && num2 <= num3 && num1 >= num3){
         print("Este es el número máximo de los tres: ${num1}");
    }

}


void main(){
   
maximo();   
   
}
